containers/proxy-httpd-image/uyuni-configure.py

```diff
@@ -5,10 +5,38 @@
 import re
 import shutil
 import yaml
+import socket
 import sys
 
 config_path = "/etc/uyuni/"
 
+def getIPs(fqdn: str) -> [str, str]:
+    addrinfo = socket.getaddrinfo(fqdn, None)
+    ipv4s = set(map(lambda r: r[4][0], filter(lambda f: f[0] == socket.AF_INET, addrinfo)))
+    ipv6s = set(map(lambda r: r[4][0], filter(lambda f: f[0] == socket.AF_INET6, addrinfo)))
+    ipv4, ipv6 = "", ""
+
+    if len(ipv4s) == 0 and len(ipv6s) == 0:
+       print("FATAL: Cannot determine proxy IPv4 nor IPv6 from FQDN {}".format(fqdn))
+       sys.exit(1)
+
+    try:
+       ipv4 = ipv4s.pop()
+       if len(ipv4s) > 0:
+          print("WARNING: Cannot determine unique IPv4 address for the proxy. TFTP sync may not work. Using IPv4 {}".format(ipv4))
```

Comment:
This script is supposed to run in container so IMO printing to stdour/stderr is what we want. That way this will appear in journald (as currently we are using systemd managed podman) or later in k8s logs and customer can then pass it to whatever logging system they use.
Or am I misunderstanding something?

Reply:
No you are right about this. However it is not possible with print to adjust the logging format for a user. Which might cause trouble on the log parsers on customer side. So for example switching from `WARNING` to `WARN` or vice-versa. As said: It might be overkill...

Reply:
Ah, I get it. Sound good, but I think we should do this in separate PR and for all container configure scripts. I created issue for it https://github.com/SUSE/spacewalk/issues/17378